### ISSUE TITLE
Anway to ask it what it knows?

### ISSUE THREAD

**Issue by Rumble25** [supabase-community/nextjs-openai-doc-search#36]
# Feature request

I think it would be neat if there was a way I could ask it what it Knows. Is there anyway I could do this, I assume i would need a way to put the data in the DB dyamically of the pages it knows?

**Comment by gregnr** [comment]
Yeah, your best bet is probably to add this to the initial prompt/instructions given to GPT. If this info is dynamic for your use case, you would need to dynamically generate it from your DB.
